easybuild/easyconfigs/c/cairo/cairo-1.14.12-GCCcore-6.4.0.eb

```diff
@@ -14,6 +14,8 @@ source_urls = ['http://cairographics.org/releases/']
 sources = [SOURCE_TAR_XZ]
 checksums = ['8c90f00c500b2299c0a323dd9beead2a00353752b2092ead558139bd67f7bf16']
 
+builddependencies = [('pkg-config', '0.29.2')]
```

Comment:
There is already a `builddependencies` array in the easyconfig file. Needs to be merged.